src/assessment.js

```diff
@@ -50,4 +50,5 @@ var q2 = [/* your answer here */];
 //
 var q5 = function fireOnAlderaan () {
   // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
Close but not quite. If you use `bind` instead of `call`, you need to explicitly invoke the function.